config/postgres/http/global_routes.json

```
@@ -0,0 +1,3 @@
+{
+  "authority": "/authority"
```

Please use a more specific route:

```suggestion
  "authority": "/security/authority"
```

or something similar.